ArcadiaParties.Data/Helpers/AutoMapperProfiles.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ArcadiaParties.Data.Abstractions.DTOs;
+using ArcadiaParties.Data.Models;
+using AutoMapper;
+using System.Linq;
+
+namespace ArcadiaParties.Data.Helpers
```

Comment:
Why is it in `Helpers` folder? Helper is some class, mostly with small static methods or constants, that can be used across the code.

Reply:
I've changed this class location